src/components/Feature.js

```diff
@@ -30,6 +49,8 @@ Feature.propTypes = {
   flagId: PropTypes.string.isRequired,
   variation: PropTypes.any,
   flags: PropTypes.object.isRequired,
+  onReady: PropTypes.func,
+  waitForLD: PropTypes.bool,
```

Comment:
this should probably default to `true`

Reply:
don't we usually put propTypes and defaultProps at the top of the class?

Reply:
This is common in libraries, likely because static fields in classes [aren't officially supported](https://github.com/tc39/proposal-static-class-features/)

Reply:
And I'd say that `waitForLD` should be false by default so that rendering can occur faster by default (with good defaults).

Reply:
I agree with @dolfelt on this one